config/config.go

```diff
@@ -46,6 +46,10 @@ type Config struct {
 	// command line arguments.
 	UntrustedEnv bool `json:"-"`
 
+	// DisableDirectoryConfiguration is used to disable the configurability of directories where we
+	// capture data. This is set via command line arguments.
+	DisableDirectoryConfiguration bool `json:"-"`
```

Comment:
I would +1 Tess's suggestion to rename to `LimitConfigurableDirectories` to make it clear, since this sounds a bit like it's the configuration to disabling directories rather than a bool.

Reply:
Oh good point. I renamed this like 5 times and none of the names were good 🙈